app/presenters/content_item_presenter.rb

```diff
@@ -8,13 +8,20 @@ def initialize(content_item)
     @format = content_item["format"]
     @locale = content_item["locale"] || "en"
     @phase = content_item["phase"]
-    @links = content_item["links"] || {}
   end
 
   def available_translations
     sorted_locales(links["available_translations"])
   end
 
+  def links
```

Comment:
Happy to make that change, but let's tidy up once this stack of branches is merged.